cmake/modules/FindPugixml.cmake

```diff
@@ -0,0 +1,39 @@
+# - Find the native Pugixml includes and library
+
```

Comment:
Since there is no *.PO so libfind_pkg_check_modules() can not be used to replace that handwritten version parsing, i guess it won't be needed.